internal/core/key.go

```diff
@@ -77,6 +79,24 @@ type DataStoreKey struct {
 
 var _ Key = (*DataStoreKey)(nil)
 
+// ViewCacheKey is a trimmed down [DataStoreKey] used for caching the results
+// of View items.
+//
+// It is stored in the format `/collection/vi/[CollectionRootID]/[ItemID]`. It points to the
+// full serialized View item.
+type ViewCacheKey struct {
+	// CollectionRootID is the Root of the Collection that this item belongs to.
+	CollectionRootID uint32
+
+	// ItemID is the unique (to this CollectionRootID) of the View item.
```

Comment:
typo: is the unique ID?

Reply:
Cheers, took me a while to figure out what you meant on re-read :)

- [ ] Add `ID` after closing of brackets